app/controllers/api/v1/api_v1_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::ApiV1Controller < Api::V1::BaseApiController
+  def index
+    json = Rails.cache.fetch('API_V1_JSON_RESPONSE', expires_in: 1.day) do
+      {
+        api: {
+          base_url: api_v1_url,
+          documentation: 'https://transit.land/documentation/datastore/api-endpoints.html',
+          swagger_description_url: 'https://github.com/transitland/transitland/issues/33', # TODO:
+          endpoints: array_of_endpoints
+        },
+      }
+    end
```

Comment:
No, this is just a homebrew list of endpoints. Enough information to get a sense for the range of endpoints available, but not enough to show query parameters, response formats, etc. That's where a full Swagger definition will be useful as a separate addition.